viite-UI/src/model/RoadNameCollection.js

```diff
@@ -36,7 +37,8 @@
                 currentRoadNameData = sortedRoadData;
                 var lastRoadName = _.last(sortedRoadData);
                 if (lastRoadName)
-                    minDate = moment(lastRoadName.startDate).add(1, 'days');
+                //TODO: Evaluate if the min date is really needed.
+                    minDate = moment(lastRoadName.startDate).subtract(yearLimit, 'years');
```

Comment:
Why do we substract five years from the startDate? Adding one day to the startDate makes more sense:
"The date need to be red until it is right  (= next day of the of the OriginalStartDate or within the next 5 years from the SysDate)."